script/setup/install-runc

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+#
+# Builds and installs runc to /usr/local/go/bin based off
+# the commit defined in vendor.conf
+#
+set -eu -o pipefail
+
+RUNC_COMMIT=$(grep opencontainers/runc ${GOPATH}/src/github.com/containerd/containerd/vendor.conf | cut -d " " -f 2)
```

Comment:
Does this match what is in RUNC.md?

Reply:
Generally no. I think we should replace RUNC.md with TOML/YAML that is readable for both machine and human. (Can be done in a separate PR)

Reply:
Well, it does match as long as PR changes are changing both locations. So far that is the case, but if we are worried about manual sync, then @AkihiroSuda might have a good idea to make a `runc` commit hash definition parseable. Of course, I assume we hope this goes away at some point as stable released `runc` will be viable for (hopefully) all releases of containerd.